### DISCUSSION TITLE
2 ruffles?

### DISCUSSION THREAD

**Discussion by Sin-Shadow-Fox** [ruffle-rs/ruffle#5950] [Q&A] [answered]
Which one of these is the legitimate ruffle extension for Firefox?
https://addons.mozilla.org/en-US/firefox/addon/ruffle_rs/
https://addons.mozilla.org/en-US/firefox/addon/ruffle-rs/

**Comment by Poopooracoocoo** [comment, marked as answer]
I don't know why there are two but you should use https://addons.mozilla.org/en-US/firefox/addon/ruffle_rs/ :)

**Reply by Sin-Shadow-Fox** [reply]
Thank you very much ^_^

**Reply by Poopooracoocoo** [reply]
The other add-on is not official
